config/locales/cy/transition_landing_page.yml

```
@@ -20,7 +20,7 @@ cy:
     video_section_description: |
       <p>Mae’n rhaid i chi weithredu nawr. Mae’r rheolau newydd yn effeithio ar ddinasyddion, busnesau a theithio i’r UE. Gwnewch yn siŵr eich bod yn barod ar gyfer ddiwedd cyfnod pontio Brexit.</p>
       <p>Gwyliwch y fideo i wybod beth fydd 2021 yn ei olygu i chi.</p>
-    video_section_link: https://www.youtube.com/watch?v=_Hyq5U4mLdQ
+    video_section_link: https://www.youtube.com/embed/Zk9bSnHwjqE
```

It seems like it needs to use the old pattern in the URL otherwise the govspeak script for the video stream won't work and will always render the placeholder instead of the video
```suggestion
    video_section_link: https://www.youtube.com/watch?v=Zk9bSnHwjqE
```

---

Ah good spot - thank you. I will amend.